privaterepo/ssh_user_list.go

```diff
@@ -0,0 +1,83 @@
+package privaterepo
+
+import (
+	"bytes"
+	"html/template"
+	"net/url"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/valuemodifier/path"
+)
+
+func ContentToSSHUserList(content string) (string, error) {
+	var err error
+
+	var rendered string
+	{
+		tmpl, err := template.New("main").Parse(content)
+		if err != nil {
+			return "", microerror.Mask(err)
+		}
+
+		v := struct {
```

Comment:
I copied that more or less from `opsctl` to just render the template. The input data does not matter as we only need the template rendered to be able to read the SSH UserList from the resulting YAML. I chose this way to not waste more time on revamping the whole thing with opsctl and the installations repo.